Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -993,9 +1000,30 @@ protected override void ProcessRecord()
                 default:
                     throw UnknownParameterSetException;
             }
+
+            string destinationBucket = DestinationBucket ?? gcsObject.Bucket;
+            string destinationObject = DestinationObjectName ?? gcsObject.Name;
+
+            if (!Force)
+            {
+                try
+                {
+                    ObjectsResource.GetRequest objGetReq =
+                        Service.Objects.Get(destinationBucket, destinationObject);
+                    objGetReq.Execute();
+                    // If destination does not exist, jump to catch statment.
```

Comment:
"jump to catch statement" sounds like you are using a goto. Perhaps:

"If destination does not exist, will throw a 404 an be ignored"